www/docs/admin/featured.php

```diff
@@ -0,0 +1,198 @@
+<style>
+.confirm-panel {
+    margin-left: 18px;
+    margin-bottom: 2em;
+}
+
+.confirm-panel input {
+    font-size: 2em;
+    font-weight: bold;
+}
+</style>
+<?php
+
+include_once '../../includes/easyparliament/init.php';
+
+$this_page = 'admin_featured';
+
+$db = new ParlDB;
+
+$PAGE->page_start();
+$PAGE->stripe_start();
+
+$out = '';
+$preview = 0;
+if (get_http_var('preview')) {
+    $preview = 1;
+    preview_featured();
+} else if ( get_http_var('confirm') ) {
+    $out = update_featured();
+}
+
+print '<div id="adminbody">';
+print $out;
+edit_featured_form();
+print '</div>';
+
+function gid_to_url($gid) {
+    if ( !$gid ) {
+        return '';
+    }
+    global $hansardmajors;
+    global $db;
+
+    $q = $db->query("SELECT major FROM hansard WHERE gid = :gid", array( ':gid' => $gid ));
+    $url_gid = fix_gid_from_db($gid);
+    $url = new \URL($hansardmajors[$q->field(0, 'major')]['page']);
+    $url->insert(array('id' => $url_gid));
+    $url = 'http://' . DOMAIN . $url->generate();
+    return $url;
+}
+
+function edit_featured_form() {
+    $featured = new MySociety\TheyWorkForYou\Model\Featured;
+    $title = $featured->get_title();
+    $gid = $featured->get_gid();
+    $related = $featured->get_related();
+    $context = $featured->get_context();
+
+    if ( get_http_var('url') ) {
+        $url = get_http_var('url');
+        $title = get_http_var('title');
+        $context = get_http_var('context');
+        $related1 = get_http_var('related1');
+        $related2 = get_http_var('related2');
+        $related3 = get_http_var('related3');
+    } else {
+        $url = gid_to_url($gid);
+        $related1 = gid_to_url($related[0]);
```

Comment:
Is the first related optional too? Looks like `gid_to_url` returns empty string if not present, so should the following two lines not bother with the `isset` either?